packages/bot-web-ui/src/pages/dashboard/announcements/announcements.tsx

```diff
@@ -0,0 +1,185 @@
+        };
+    }, []);
+
+    const handleAnnounceSubmit = (data: Record<string, boolean>) => {
+        setAmountAnnounce(data);
+        localStorage?.setItem('bot-announcements', JSON.stringify(data));
+    };
+
+    const announcements = [
+        {
+            icon: <IconAnnounce announce={is_active_announce_1} />,
+            title: (
+                <TitleAnnounce title={localize('Accumulators is now on Deriv Bot')} announce={is_active_announce_1} />
+            ),
+            message: (
+                <MessageAnnounce
+                    message={localize('Boost your trading strategy with Accumulators.')}
+                    date='20 July 2024 00:00 UTC'
+                    announce={is_active_announce_1}
+                />
+            ),
+            buttonAction: () => {
+                setIsAnnounceDialogOpen(true);
+                setIsOpenAnnounceList(prev => !prev);
+                handleAnnounceSubmit({ ...amountAnnounce, announce_1: false });
+            },
+            actionText: '',
+        },
+    ];
+
+    useEffect(() => {
+        let data: Record<string, boolean> | null = null;
+        data = JSON.parse(localStorage.getItem('bot-announcements') ?? '{}');
+
+        if (data && Object.keys(data).length !== 0) {
+            setAmountAnnounce(data);
+        } else {
+            const obj_announcements = Object.fromEntries(
+                Array.from({ length: announcements.length }, (_, i) => [`announce_${i + 1}`, true])
+            );
+            setAmountAnnounce(obj_announcements);
+            localStorage?.setItem('bot-announcements', JSON.stringify(obj_announcements));
+        }
+    }, [announcements.length]);
+
+    const handleOnCancelAccumulator = () => {
+        handleTabChange(DBOT_TABS.TUTORIAL);
+    };
+
+    const handleOnConfirmAccumulator = async () => {
+        handleTabChange(DBOT_TABS.BOT_BUILDER);
+        const strategy_xml = await import(
+            /* webpackChunkName: `[request]` */ '@deriv/bot-skeleton/src/scratch/xml/main.xml'
+        );
+        const strategy_dom = Blockly.utils.xml.textToDom(strategy_xml.default);
+        const modifyFieldDropdownValues = (name: string, value: string) => {
+            const name_list = `${name.toUpperCase()}_LIST`;
+            const el_blocks = strategy_dom?.querySelectorAll(`field[name="${name_list}"]`);
+
+            el_blocks?.forEach((el_block: HTMLElement) => {
+                el_block.textContent = value;
+            });
+        };
+        modifyFieldDropdownValues('tradetypecat', 'accumulator');
+
+        const { derivWorkspace: workspace } = Blockly;
+
+        await load({
+            block_string: Blockly.Xml.domToText(strategy_dom),
+            file_name: 'Strategy with accumulator trade type',
+            workspace,
+            from: save_types.UNSAVED,
+            drop_event: null,
+            strategy_id: null,
+            showIncompatibleStrategyDialog: null,
+        });
+    };
```

Comment:
@maryia-matskevich-deriv if we put these specific logics in this announcement component then moving forward this file is will be huge and not maintainable. Can we think of a different approach? Announce should just render announcements from a config file and we can keep the announcement modal here but the modal content should come from the config. Let me know if you want to discuss about this approach.

Reply:
@shafin-deriv The modal content has already come from the config. Do you worry only about this method? 
Let's discuss it.

Reply:
I am actually not worried about this method but I am concerned that if we go by this approach then having 9 or 10 more notifications would flood this component with specific functions for each. So, it would be better to think of a better way to handle this now than later 😃

Reply:
✅ refactored as you suggested